go.mod

```
@@ -120,4 +120,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace github.com/distribution/distribution/v3 => github.com/openshift/docker-distribution/v3 v3.0.0-20240826125755-2799294adc70
+replace github.com/distribution/distribution/v3 => github.com/mpryc/docker-distribution/v3 v3.0.0-20240826125755-2799294adc70
```

```suggestion
replace github.com/distribution/distribution/v3 => github.com/mpryc/docker-distribution/v3 v3.0.0-20240926170138-55b4196b7cd5
```
hash from https://github.com/openshift/docker-distribution/pull/44